package.json

```diff
@@ -45,13 +45,16 @@
     "@rollup/plugin-node-resolve": "^8.0.1",
     "@rollup/plugin-replace": "^2.3.3",
     "@testing-library/react": "^10.4.8",
+    "@types/chai": "^4.2.3",
+    "@types/chai-dom": "^0.0.10",
     "@types/enzyme": "^3.10.5",
     "@types/enzyme-adapter-react-16": "^1.0.6",
     "@types/jest": "^26.0.14",
     "@types/mocha": "^8.0.3",
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.25",
     "@types/react-dom": "^16.9.5",
+    "@types/sinon": "^9.0.8",
```

Comment:
ohhhh it would be so good to move away of this along with chai...

Reply:
We will host components here that won't need to be end-to-end tested as much as the data grid need to (simpler).
I think that sinon and chai are here to stay for a couple of years:

- Jest doesn't allow to run tests in the browsers, it wasn't designed nor optimized for the use cases for a UI library. It doesn't seem that they will allow execution outside of JSDOM in the future. Maybe one day. This definitely has the potential to make a better test runner than mocha+karma. As for sinon and chai, it probably doesn't really matter what we use. We aim to write end-to-end tests, mocking shouldn't be critical.
- We have experienced that running and controlling the tests in the same JavaScript VM yields a superior DX for the simplest test cases and components, like most of what TreeView needs or a Combobox. Puppeteer run and control test in two different JavaScript VM, which can harm the quality of the assertions & performance. Maybe it's possible to reach a similar level of DX, but we aren't here yet with the tooling, it's going to take time.